inst/examples/AA-AA-000-0000/config/2024-09-26/config.yaml

```diff
@@ -0,0 +1,258 @@
+      Raw_DATACHG:
+        db: clindata
+        table: edc_data_points
+      Raw_QUERY:
+        db: clindata
+        table: edc_queries
+  Mapped:
+    db:
+      local: ./data/2024-09-26/Mapped/{ID}.csv
+      s3: s3://AA-AA-000-0000/data/2024-09-26/Mapped/{ID}.parquet
+    domains:
+      Mapped_STUDY:
+        db: local
+        table: ./data/2024-09-26/Mapped/STUDY.csv
+      Mapped_SITE:
+        db: local
+        table: ./data/2024-09-26/Mapped/SITE.csv
+      Mapped_COUNTRY:
+        db: local
+        table: ./data/2024-09-26/Mapped/COUNTRY.csv
+      Mapped_SUBJ:
+        db: local
+        table: ./data/2024-09-26/Mapped/SUBJ.csv
+      Mapped_ENROLL:
+        db: local
+        table: ./data/2024-09-26/Mapped/ENROLL.csv
+      Mapped_STUDCOMP:
+        db: local
+        table: ./data/2024-09-26/Mapped/STUDCOMP.csv
+      Mapped_SDRGCOMP:
+        db: local
+        table: ./data/2024-09-26/Mapped/SDRGCOMP.csv
+      Mapped_AE:
+        db: local
+        table: ./data/2024-09-26/Mapped/AE.csv
+      Mapped_LB:
+        db: local
+        table: ./data/2024-09-26/Mapped/LB.csv
+      Mapped_PD:
+        db: local
+        table: ./data/2024-09-26/Mapped/PD.csv
+      Mapped_DATAENT:
+        db: local
+        table: ./data/2024-09-26/Mapped/DATAENT.csv
+      Mapped_DATACHG:
+        db: local
+        table: ./data/2024-09-26/Mapped/DATACHG.csv
+      Mapped_QUERY:
+        db: local
+        table: ./data/2024-09-26/Mapped/QUERY.csv
+  Analysis:
+    db:
+      local: ./data/2024-09-26/Analysis/{ID}/*.csv
+      s3: s3://AA-AA-000-0000/data/2024-09-26/Analysis/{ID}/*.parquet
+    domains:
+      Analysis_Input:
+        db: local
+        table: ./data/2024-09-26/Analysis/{ID}/Analysis_Input.csv
+      Analysis_Summary:
+        db: local
+        table: ./data/2024-09-26/Analysis/{ID}/Analysis_Summary.csv
+  Reporting:
+    db:
+      local: ./data/2024-09-26/Reporting/{ID}.csv
+      s3: s3://AA-AA-000-0000/data/2024-09-26/Reporting/{ID}.parquet
+    domains:
+      Reporting_Results:
+        db: local
+        table: ./data/2024-09-26/Reporting/Results.csv
+      Reporting_Metrics:
+        db: local
+        table: ./data/2024-09-26/Reporting/Metrics.csv
+      Reporting_Groups:
+        db: local
+        table: ./data/2024-09-26/Reporting/Groups.csv
+      Reporting_Bounds:
+        db: local
+        table: ./data/2024-09-26/Reporting/Bounds.csv
+dependencies:
+- id: gsm
+  repo: Gilead-BioStats/gsm
+  version: dev
+- id: grail
+  repo: Gilead-BioStats/grail
+  version: 1.0.0
+- id: gsm.app
+  repo: Gilead-BioStats/gsm.app
+  version: 1.0.0
+- id: gsm.endpoints
+  repo: Gilead-BioStats/gsm.endpoints
+  version: 1.0.0
+modules:
+- id: report_kri_site
+  repo: Gilead-BioStats/gsm
+  path: inst/workflow/reports/report_kri_site.yaml
+  version: dev
+  package: gsm
+  file: workflow/reports/report_kri_site.yaml
+metrics:
+- id: kri0001
+  repo: Gilead-BioStats/gsm
+  path: inst/workflow/metrics/kri0001.yaml
```

Comment:
I wonder if we really need this. Feel like we could just have almost all of this in the workflow yaml header. And then just use standard paths ... Maybe add it in the future if we need customization